src/modules/services/submodule/nodes/views/ViewServicesNodes.vue

```diff
@@ -100,21 +95,18 @@ export default {
     const currentNode = computed(() =>  NetworkStateUtils.buildAPIEndpointURL(networkState.selectedAPIEndpoint))
     const blockHeight = computed(() => networkState.currentNetworkProfileConfig.chainHeight);
 
-    const makeNodeSelection = (e) => {
-      if(e != networkState.selectedAPIEndpoint){
+    const makeNodeSelection = (endpoint) => {
+      if(endpoint != networkState.selectedAPIEndpoint){
         showSelectTitle.value = true;
-        NetworkStateUtils.updateChainNode(e)
-        /* stopListening(); */
-       
-       /*  startListening(walletSession.accounts); */
+        NetworkStateUtils.updateChainNode(endpoint);
+        WalletUtils.refreshAllAccountDetails(walletState.currentLoggedInWallet, networkState.currentNetworkProfile);
         WalletUtils.getTotalBalanceWithCurrentNetwork();
```

Comment:
don't need it

Reply:
Are you referring to `WalletUtils.getTotalBalanceWithCurrentNetwork();`?